.vib/alertmanager/goss/vars.yaml

```diff
@@ -0,0 +1,14 @@
+binaries:
+  - alertmanager
+  - amtool
+directories:
+  - paths:
+      - /opt/bitnami/alertmanager
+      - /opt/bitnami/alertmanager/conf
+files:
+  - paths:
+      - /opt/bitnami/alertmanager/conf/config.yml
```

Comment:
You can easily test the application can start up correctly by using this file, as you indicate in the PR description. To bypass the command from running forever, you can make use of `timeout`:

```
$ timeout --preserve-status 5 /opt/bitnami/alertmanager/bin/alertmanager --config.file=/opt/bitnami/alertmanager/conf/config.yml --storage.path=/opt/bitnami/alertmanager/data
ts=2023-05-08T15:51:37.381Z caller=main.go:240 level=info msg="Starting Alertmanager" version="(version=0.25.0, branch=HEAD, revision=258fab7cdd551f2cf251ed0348f0ad7289aee789)"
ts=2023-05-08T15:51:37.381Z caller=main.go:241 level=info build_context="(go=go1.19.4, user=root@abe866dd5717, date=20221222-14:51:36)"
ts=2023-05-08T15:51:37.381Z caller=cluster.go:185 level=info component=cluster msg="setting advertise address explicitly" addr=172.17.0.2 port=9094
ts=2023-05-08T15:51:37.384Z caller=cluster.go:681 level=info component=cluster msg="Waiting for gossip to settle..." interval=2s
ts=2023-05-08T15:51:37.416Z caller=coordinator.go:113 level=info component=configuration msg="Loading configuration file" file=/opt/bitnami/alertmanager/conf/config.yml
ts=2023-05-08T15:51:37.417Z caller=coordinator.go:126 level=info component=configuration msg="Completed loading of configuration file" file=/opt/bitnami/alertmanager/conf/config.yml
ts=2023-05-08T15:51:37.462Z caller=tls_config.go:232 level=info msg="Listening on" address=[::]:9093
ts=2023-05-08T15:51:37.462Z caller=tls_config.go:235 level=info msg="TLS is disabled." http2=false address=[::]:9093
ts=2023-05-08T15:51:39.384Z caller=cluster.go:706 level=info component=cluster msg="gossip not settled" polls=0 before=0 now=1 elapsed=2.000210082s
ts=2023-05-08T15:51:42.328Z caller=main.go:583 level=info msg="Received SIGTERM, exiting gracefully..."
ts=2023-05-08T15:51:42.329Z caller=cluster.go:690 level=info component=cluster msg="gossip not settled but continuing anyway" polls=2 elapsed=4.944808208s
I have no name!@c376dbc13f13:/opt/bitnami/alertmanager$ echo $?
0
```